### PR TITLE
Adapt snappy-raw compression read

### PR DESCRIPTION
**Pull request by Linary**
Change-Id: I86d391086dd79cdee4a2f01fa45c669486834fb1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -105,10 +105,29 @@
...
+<!--                <exclusion>-->
+<!--                    <groupId>com.sun.jersey</groupId>-->
+<!--                    <artifactId>jersey-core</artifactId>-->
+<!--                </exclusion>-->
+<!--                <exclusion>-->
+<!--                    <groupId>com.sun.jersey</groupId>-->
+<!--                    <artifactId>jersey-server</artifactId>-->
+<!--                </exclusion>-->
+<!--            </exclusions>-->
+<!--        </dependency>-->
```
Comment (javeme) [inline pom.xml]: remove

File: src/test/java/com/baidu/hugegraph/loader/test/functional/FileLoadTest.java [hunk 2]
```diff
@@ -1561,10 +1561,7 @@ public void testLZMACompressFile() {
         Assert.assertEquals(1, vertices.size());
     }
 
-    /**
-     * Didn't find a way to generate the compression file using code
-     */
-    //@Test
+//    @Test
```
Comment (javeme) [inline src/test/java/com/baidu/hugegraph/loader/test/functional/FileLoadTest.java]: uncomment

Comment (Linary) [reply src/test/java/com/baidu/hugegraph/loader/test/functional/FileLoadTest.java]: The hadoop 2.8.5 seems difficult to be compatible

**Comment by javeme** [comment]
ci-failed

**Review by javeme — APPROVED** [review]

**Review by houzhizhen — APPROVED** [review]

**Pull request merged**